pyramid/tweens.py

```diff
@@ -42,7 +42,7 @@ def excview_tween(request):
             provides = providedBy(exc)
             try:
                 response = _call_view(
-                    registry,
+                    request.registry,
```

Comment:
Again, I guess the issue here is that the registry eventually retains the `excview_tween` and the `excview_tween` retains the registry via the closure, but this is all cleaned up via GC in general and likely occurs in many other parts of code.

What practical problem is this solving that I should accept this change? Is the submitted integration test going to prevent me from writing this code again later?